src/accidental.js

```diff
@@ -56,7 +56,7 @@ export class Accidental extends Modifier {
         }
         prevNote = note;
       }
-      if (stave !== null) {
+      if (stave) {
```

Comment:
that is not the same as:
```javascript
if (stave !== null)
```
I would be more careful with these changes

Reply:
That was done on purpose. I believe that this is the right way to check that an `object` is not `null` or `undefined`

Reply:
Yes, this is fine. Thanks.